### PR TITLE
Add support for aarch64 python 3.9

### PR DESCRIPTION
**Pull request by mathbunnyru**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

@conda-forge-admin, please rerender

Fix: https://github.com/conda-forge/bottleneck-feedstock/issues/35

### CODE REVIEW

**Comment by mathbunnyru** [comment]
I copy-pasted aarch64 file for python3.8 and changed the version - is there anything else I should do?

**Comment by mathbunnyru** [comment]
I bumped numpy to 1.19 for this python version - is it ok? I've seen it used in other places as well.

**Comment by mathbunnyru** [comment]
Travis Ci is misbehaving

**Comment by mathbunnyru** [comment]
@jakirkham @msarahan @ocefpaf @qwhelan @wesm may I ask you to take a look please?

File: .ci_support/linux_aarch64_python3.9.____cpython.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+BUILD:
+- aarch64-conda_cos7-linux-gnu
```
Comment (jakirkham) [inline .ci_support/linux_aarch64_python3.9.____cpython.yaml:2]: Why is this being added manually? Did you try [re-rendering]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#dev-update-rerender )?

Comment (mathbunnyru) [reply .ci_support/linux_aarch64_python3.9.____cpython.yaml:2]: Thanks, I will take a look at what that means and try to re-render.
Sorry, I'm a beginner in conda-forge maintenance.

Comment (jakirkham) [reply .ci_support/linux_aarch64_python3.9.____cpython.yaml:2]: xref: https://github.com/conda-forge/bottleneck-feedstock/issues/37

**Comment by jakirkham** [comment]
PR ( https://github.com/conda-forge/bottleneck-feedstock/pull/38 ) should handle this, but there is a lingering question for one of the maintainers

**Comment by mathbunnyru** [comment]
Thank you! Hope this will be merged soon :)

**Pull request closed without merging**